<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="q_rad_large">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Large</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="q_rad_midium">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Midium</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="q_rad_small">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>161</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Small</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="q_btn_home">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>241</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>Select Pressure</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="q_rad_Level1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Level1</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="q_rad_Level2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Level2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="q_rad_Level3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Level3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="q_chkb_washonly">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Wash Only</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="q_btn_stop">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>450</y>
      <width>160</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
